### DISCUSSION TITLE
How to get irregular compartment grid or other non standard configurations

### DISCUSSION THREAD

**Discussion by Le0Michine** [kennetek/gridfinity-rebuilt-openscad#35] [Q&A]
I see some interesting variations for bin compartments configuration on this gif but it seems like the only available option to configure right now is rectangular grid. Is there a way to get irregular grid or grid with some round holes like on that gif in the readme.md?

![image](https://user-images.githubusercontent.com/2707008/206948967-a44e9448-a468-4f95-b531-dcac8625696f.png)

**Comment by kennetek** [comment]
Like I answered in discussion https://github.com/kennetek/gridfinity-rebuilt-openscad/discussions/29, the models shown in the gifs are in the example section at the bottom of the `gridfinity-rebuilt-bins.scad` file. The one with the holes is the fourth example. The equally spaced grid is the default because that is the typical use case. If you are interested in the more advanced modules, there is full documentation on the [website](https://kennetek.github.io/gridfinity-rebuilt-openscad/). Hope that helps!

**Reply by Le0Michine** [reply]
Completely missed the commented parts, thanks! Will take a closer look
